### DISCUSSION TITLE
Five region active controller

### DISCUSSION THREAD

**Discussion by Nakhchi** [xcompact3d/Incompact3d#275] [Q&A] [answered]
Hi,

In the "NREL-5MW.turb"  there are 3 different operation flags.
1: Prescribed velocity, 2: five region active controller, 3: reading from a list

What is the definition of "Five region active controller" ? 

And is this the "axial velocity" or "rotational velocity"?

Many thanks,
Mahdi

**Comment by nbeb** [comment, marked as answer]
Hi Mahdi, 

Option 2 is the implementation of the controller described in: https://www.nrel.gov/docs/fy09osti/38060.pdf 
Can you further explain what you mean by: Is this the "axial velocity" or "rotational velocity"?

Best, 
Nikos
